.github/workflows/ci.yml

```diff
@@ -101,6 +101,14 @@ jobs:
         flags: unit
         fail_ci_if_error: false
 
+  auto-merge:
```

Comment:
This should probably only run on a special PR target event.

Reply:
How would we go about doing that? The auto-merge examples just look like this.

I guess it'd be nice to only run the action if the author is dependabot, but I don't see any options like that.

Reply:
You could use `if`-clauses to decide whether to run a certain job. Also, I'm pretty sure that https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target is necessary. Other contexts may not have access to the secrets.

Reply:
Does that look about right now?

Reply:
almost